app/serializers/invite_serializer.rb

```diff
@@ -1,21 +1,39 @@
 # frozen_string_literal: true
 
 class InviteSerializer < ApplicationSerializer
-
-  attributes :email, :updated_at, :redeemed_at, :expired, :user
+  attributes :id, :email, :updated_at, :redeemed_at, :expired, :user, :invite_source
 
   def include_email?
-    options[:show_emails] && !object.redeemed?
+    options[:type] == "pending" && options[:show_emails] && !object.redeemed?
+  end
+
+  def include_redeemed_at?
+    options[:type] == "redeemed"
```

Comment:
It'll probably be a good idea to move `redeemed` and `pending` into constants so we can avoid repeatedly hardcoding the strings in multiple places making it hard to change in the future.

Reply:
Since we now have two different serializers for pending/redeemed no need to check for type anymore.